openhtf/io/user_input.py

```diff
@@ -24,14 +24,20 @@
 
 import collections
 import functools
+import logging
+import platform
 import select
 import sys
-import termios
 import threading
 import uuid
 
 import gflags
 
+if platform.system() != 'Windows':
+  import termios
```

Comment:
As discussed in meeting, we wanted to consider using pyreadline as a better cross-platform prompting alternative.